<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>244</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>244</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MediaPlayerP_21</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/ico/ico/note.ico</normaloff>:/ico/ico/note.ico</iconset>
  </property>
  <widget class="QLabel" name="labelFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Song file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrev">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderVolume">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolume">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelProgress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuration">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
